CMakeLists.txt

```
@@ -29,6 +30,9 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules")
 include(TargetArch)
 detect_target_architecture(_TARGET_ARCH)
 
+CHECK_INCLUDE_FILE("<getopt.h>" HAVE_GETOPT_H)
+cmake_dependent_option(BASE64_BUILD_CLI "Build the cli for encoding and decoding" ON "HAVE_GETOPT_H" ON)
```

```suggestion
cmake_dependent_option(BASE64_BUILD_CLI "Build the cli for encoding and decoding" ON "HAVE_GETOPT_H" OFF)
```
If `<getopt.h>` is not available (like on windows), we can't build th CLI. Therefore we need to force it off.

However, the include check is borked (see my other comment).